x/cheqd/types/stateValue.go

```diff
@@ -24,7 +24,7 @@ func NewStateValue(data StateValueData, metadata *Metadata) (*StateValue, error)
 	return &StateValue{Data: any, Metadata: metadata}, nil
 }
 
-func NewMetadata(ctx sdk.Context) Metadata {
+func NewMetadataFromContext(ctx sdk.Context) Metadata {
```

Comment:
Our DID specification states the versionId is the transaction hash, not a base64 encoded version of the tx hash. I'm more inclined to keep it as just the tx hash, for instance, because that can easily be plugged into a block explorer to pull up that transaction. Do you agree?

Reply:
I would say we have to encode it the same way as it's done in Tendermint/block explorer (cause it's a byte array, it has to be encoded). @Toktar If I'm not mistaken there was a ticket for it.